core/src/main/java/dk/alexandra/fresco/framework/network/KryoNetNetwork.java

```diff
@@ -114,7 +114,9 @@ public NaiveListener(Map<Integer, BlockingQueue<byte[]>> queue) {
         if (pPort == null) {
           continue;
         }
-        if (pHost.equals(hostname) && pPort == port) {
+        // The special case of localhost == 127.0.0.1 enables windows users to play along.
+        if ((pHost.equals(hostname) || (pHost.equals("localhost") && hostname.equals("127.0.0.1")))
+            && pPort == port) {
           return queue.get(i);
```

Comment:
This has the disadvantage of not working in a complex network setup. The servers from BDbS have two network interfaces and host resolving is complicated for the application. Especially if it is run in a docker container.

Might I suggest a different strategy (or none at all) for checking the host name and party identifying? A simple version could just to forward your own contact details - given the security of the protocols this handshake only need to be correct, not unbreakable